### PR TITLE
Add ProductBadge component

### PR DESCRIPTION
**Pull request by dloranc**
This PR adds a ProductBadge component.

image

### CODE REVIEW

File: components/product/ProductBadge.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+				return Math.round(((regular - sale) / regular) * 100)
+			}
+
+			return 0
+		}
+	}
+}
+</script>
+
+<style lang="sass">
```
Comment (jan-gajdowicz) [inline components/product/ProductBadge.vue:43]: Please use Oswald font, as in current page:

`.tile .label {
    position: absolute;
    top: 0;
    left: 0;
    border-radius: 0;
    font-family: oswald,sans-serif;
    color: #fff;
    text-transform: uppercase;
    font-weight: 200;
    font-size: 14px;
    padding: 4px 10px;
}`

**Review by jan-gajdowicz — COMMENTED** [review]
Please add component to single-product view, as in current page :)

**Comment by dloranc** [comment]
Changes from review implemented.

**Review by jan-gajdowicz — APPROVED** [review]
GJ! :champagne:

**Pull request merged**